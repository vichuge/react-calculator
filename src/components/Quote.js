import React from 'react';

const Quote = () => (
  <h2 className="quote-text">
    Mathematics is not about numbers, equations, computations, or algorithms:
    it is about understanding. â€“William Paul Thurston
  </h2>
);

export default Quote;
