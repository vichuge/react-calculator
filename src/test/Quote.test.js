import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('return display', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
