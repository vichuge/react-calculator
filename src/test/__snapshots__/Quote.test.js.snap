// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`return display 1`] = `
<h2
  className="quote-text"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
</h2>
`;
